core/src/main/resources/hudson/PluginManager/_table.css

```diff
@@ -87,6 +87,7 @@ time {
 
 #plugins tr.unavailable {
   background-color: #f4f4f4;
```

Comment:
Why keep this line?

Reply:
For IE11 compatibility

We have a plugin that runs over the core less files and automatically adds this. but it doesn’t run on the adjuncts